tests/queries/0_stateless/03234_get_setting_or_default.sql

```diff
@@ -0,0 +1,26 @@
+SET custom_a = 'value_a';
+SET custom_b = 'value_b';
+SET custom_c = null;
+SET custom_d = 5;
+
+
```

Comment:
What's the point of so many empty lines?

Reply:
Done, thanks!. Removed